AdaLoverNFT

```diff
@@ -0,0 +1,13 @@
+{
+    "project": "AdaLoverNFT",
```

Comment:
"tags": [
    "AdaLover",
 ],

Reply:
Trailing commas are not possible in JSON format